api/v1alpha5/virtualmachine_hardware_types.go

```diff
@@ -36,6 +75,26 @@ const (
 	SCSIControllerTypeLsiLogicSAS     SCSIControllerType = "LsiLogicSAS"
 )
 
+// MaxSlots returns the maximum number of devices per SCSI controller type.
+// Note: The controller itself occupies one slot (unit number 7).
+//
+// PVSCSI supports 64 devices per controller starting vSphere 6.7+.
+// Since min supported vSphere version is >= 8.0, this is a safe assumption.
+//
+// For all these sub-types, the max slots is 1 less than the capacity
```

Comment:
Strunk & White rules, as well as the AP and Chicago Style manuals, indicate to spell out a number less than 10. For example:

```golang
SCSI controllers occupy a slot on their own bus, unit number seven. That is why
a SCSI controller supports 63 disks, even though the controller actually supports
64 devices. The controller itself takes up one of those device slots.
```